deployment/entity-service/requirements.yaml

```diff
@@ -8,6 +8,6 @@ dependencies:
     repository: https://kubernetes-charts.storage.googleapis.com
     condition: provision.minio
   - name: postgresql
-    version: 8.3.0
-    repository: https://kubernetes-charts.storage.googleapis.com
+    version: 8.9.1
```

Comment:
How often would you recommend us to check the version of dependent charts? Would there be similar dependency bot for charts?

Reply:
I would do it anytime you change the kubernetes deployment template.